src/main/target/NAZE/target.h

```diff
@@ -157,7 +157,9 @@
 // USART2, PA3
 #define BIND_PIN                PA3
 
+#if !defined(BEEBRAIN)
```

Comment:
Yes, but if someone adds a Naze-based brushed target in future this will cover it.